### ISSUE TITLE
[macOS Safari 13 Extension] Google doesn't work with Krypton. 

### ISSUE THREAD

**Issue by pugthelouie** [kryptco/kr-u2f#45]
Hello! It seems when trying to register my Krypton "security key" with Google, it just replies "Remove your security key and try again." 

Any insight on what happened?

![odd](https://foxes-are.best/A3nQ8EW.png
)

**Comment by agrinman** [comment]
Confirmed this is an issue. 
Work around: register your key on chrome/firefox, and then it will work in safari for authentication.

Long term, Google doesn't allow u2f registration on safari -- they enforce webauthn only which may need adjustments on the krypton mobile app.

**Comment by agrinman** [comment]
Does anyone have a hardware u2f/fido2 key that can test registration on Google?
I have a hunch that registration is broken of Safari for any security key...

**Comment by pugthelouie** [comment]
I am planning to get one soon... however your workaround worked perfectly. I’ll check back in a while when I make the purchase to see if you still need help.

**Comment by agrinman** [comment]
I have many, just they are not with me currently :/

**Comment by agrinman** [comment]
Pretty sure this is a google thing, https://webauthndemo.appspot.com works fine for me (it's a demo built by google). Curious if the hardware token works...

**Comment by lumaxis** [comment]
@agrinman I just tested it with my [Solo](https://solokeys.com/) key in Safari and it worked just fine.

**Comment by agrinman** [comment]
@lumaxis you tested registration right?

**Comment by lumaxis** [comment]
Yes, sorry. Realized that my comment is a bit ambiguous just after posting. Edited it for clarity 🙂

**Comment by agrinman** [comment]
Ok thanks, that’s good then! Hopefully they’re not blocking because of something like attestation. Will debug further!

**Comment by mkexc** [comment]
Any updates on this? I have this bug too.

**Comment by agrinman** [comment]
Not yet...still an open issue. I'd suggest registering your krypton key on the chrome browser. Authentication works just fine on Safari.
